lib/svrquery/protocol/titanfall/query_test.go

```diff
@@ -41,31 +41,40 @@ var (
 			Map:             "mp_rr_desertlands_64k_x_64k",
 		},
 		PerformanceInfo: PerformanceInfo{
-			AverageFrameTime:       0,
-			MaxFrameTime:           0,
-			AverageUserCommandTime: 0,
-			MaxUserCommandTime:     0,
+			AverageFrameTime:       1,
+			MaxFrameTime:           2,
+			AverageUserCommandTime: 3,
+			MaxUserCommandTime:     4,
 		},
 		MatchState: MatchState{
-			Phase:            2,
-			MaxRounds:        1,
-			RoundsWonIMC:     0,
-			RoundsWonMilitia: 0,
-			TimeLimit:        1800,
-			TimePassed:       0,
-			MaxScore:         50,
+			MatchStateV2: MatchStateV2{
```

Comment:
Not sure why this change as they look identical, could you clarify?

Reply:
MatchState supports an additional TeamsLeftWithPlayersNum field (which is set below for the v7 test) to enable this to still process older requests (e.g. the keyed test) I have updated the MatchState object to include it, and embedded the older MatchStateV2 inside.